packages/compose-tunnel-agent/src/ssh/tunnel-set.ts

```diff
@@ -0,0 +1,50 @@
+import net from 'net'
+import { RunningService } from '../docker'
+
+interface Forward {
+  service: RunningService
+  host: string
+  port: number
+  sockets: Set<net.Socket>
+  forwardRequestId: string
+}
+
+const forwardKey = (forwardRef: string) => forwardRef.replace(/^\/|#.*/g, '')
+
+interface TunnelSet {
```

Comment:
is this interface needed? it's not exported

Reply:
yup, otherwise you can't refere to it in the difference function signature